share/completions/tmutil.fish

```diff
@@ -1,61 +1,136 @@
-complete -c tmutil -n __fish_use_subcommand -a listlocalsnapshotdates -d 'List the creation dates of all local Time Machine snapshots'
-complete -r -c tmutil -n __fish_use_subcommand -a listlocalsnapshots -d 'List local Time Machine snapshots of the specified volume'
-complete -f -c tmutil -n __fish_use_subcommand -a listbackups -d 'Print paths for all snapshots'
-complete -f -c tmutil -n __fish_use_subcommand -a localsnapshot -d 'Create new local Time Machine snapshot of APFS volume in TM backup'
-complete -f -c tmutil -n __fish_use_subcommand -a machinedirectory -d 'Print the path to the current machine directory'
-complete -f -c tmutil -n __fish_use_subcommand -a removedestination -d 'Removes a backup destination'
-complete -f -c tmutil -n __fish_use_subcommand -a removeexclusion -d 'Remove an exclusion'
-complete -f -c tmutil -n '__fish_seen_subcommand_from removeexclusion' -s v -d 'Volume exclusion'
-complete -f -c tmutil -n '__fish_seen_subcommand_from removeexclusion' -s p -d 'Path exclusion'
-complete -f -c tmutil -n __fish_use_subcommand -a restore -d 'Restore an item'
-complete -r -c tmutil -n '__fish_seen_subcommand_from restore' -s v
-complete -r -c tmutil -n __fish_use_subcommand -a setdestination -d 'Set a backup destination'
-complete -f -c tmutil -n '__fish_seen_subcommand_from setdestination' -s a -d 'Add to the list of destinations'
-complete -f -c tmutil -n '__fish_seen_subcommand_from setdestination' -s p -d 'Enter the password at a non-echoing interactive prompt'
-complete -f -c tmutil -n __fish_use_subcommand -a snapshot -d 'Create new local Time Machine snapshot'
-complete -f -c tmutil -n __fish_use_subcommand -a startbackup -d 'Begin a backup if one is not already running'
-complete -f -c tmutil -n '__fish_seen_subcommand_from startbackup' -s a -l auto -d 'Automatic mode'
-complete -f -c tmutil -n '__fish_seen_subcommand_from startbackup' -s b -l block -d 'Block until finished'
-complete -f -c tmutil -n '__fish_seen_subcommand_from startbackup' -s r -l rotation -d 'Automatic rotation'
-complete -r -c tmutil -n '__fish_seen_subcommand_from startbackup' -s d -l destination -d 'Backup destination'
-complete -f -c tmutil -n __fish_use_subcommand -a stopbackup -d 'Cancel a backup currently in progress'
-complete -r -c tmutil -n __fish_use_subcommand -a thinlocalsnapshots -d 'Thin local Time Machine snapshots for the specified volume'
-complete -r -c tmutil -n __fish_use_subcommand -a uniquesize -d 'Analyze the specified path and determine its unique size'
-complete -r -c tmutil -n __fish_use_subcommand -a verifychecksums -d 'Verify snapshot'
-complete -f -c tmutil -n __fish_use_subcommand -a version -d 'Print version'
-
-complete -f -c tmutil -n '__fish_seen_subcommand_from destinationinfo isexcluded compare' -s X -d 'Print as XML'
+function __destination_ids
+    for line in $(tmutil destinationinfo)
+        if test -n "$(string match '*===*' $line)"
```

Comment:
`string match -q`, no `test`.

Also use `--` before the variable, or it'll break on lines starting with `-`.

So:

```fish
if string match -q '*===*' -- $line
```

Same goes for the other `string` uses.

Reply:
Oh that's much nicer. Thanks!